pkg/apis/sources/v1alpha1/ping_types.go

```diff
@@ -51,6 +52,16 @@ var (
 	_ apis.HasSpec = (*PingSource)(nil)
 )
 
+const (
```

Comment:
This is already here: https://github.com/knative/eventing/blob/edbcab434648360c00fd2b1a1e13dc570913ca2b/pkg/apis/sources/v1alpha1/ping_lifecycle.go#L55-L63